agent/agent-bootstrap/src/test/java/com/microsoft/applicationinsights/agent/bootstrap/diagnostics/status/StatusFileTests.java

```diff
@@ -146,7 +146,7 @@ private void runWriteFileTest(boolean enabled) throws Exception {
     assertThat(tempFolder.isDirectory()).isTrue();
     assertThat(tempFolder.list()).isEmpty();
 
-    StatusFile.directory = tempFolder.getAbsolutePath();
+    StatusFile.logDir = tempFolder.getAbsolutePath();
```

Comment:
can you check that the tests in this file pass? they're `@Disabled` so don't run on CI, but worth verifying them locally since lots of related changes in this PR

Reply:
green now.

Reply:
fixed envvars running on diff threads and caused the test failure. i don't think it was green in the past. we need to enable the test without using tempfolder in the future.